### PR TITLE
Update contact contributor list

### PR DESCRIPTION
**Pull request by adamscott**
This PR updates the contributor list in the contact page as of April 23, 2023.

### CODE REVIEW

File: pages/contact.html [hunk 1]
```diff
@@ -164,7 +170,7 @@ <h4>Developers:</h4>
 		<li>Guilherme Felipe de C. G. da Silva (guilhermefelipecgs)</li>
 		<li>Hanif Bin Ariffin (hbina)</li>
 		<li>Haoyu Qiu (timothyqiu)</li>
-		<li>HP van Braam (hpvb)</li>
```
Comment (coppolaemilio) [inline pages/contact.html:167]: I believe this is how HP prefers their name to be written, so I wouldn't change this line.

**Comment by YuriSizov** [comment]
Normally, Remi would update this page, so we should check with him what threshold would he use to include people (and how to format their names and such). cc @akien-mga

**Comment by akien-mga** [comment]
As long as this was directly synced from https://github.com/godotengine/godot/blob/master/AUTHORS.md, it should be good to go.

**Comment by akien-mga** [comment]
Updated to include https://github.com/godotengine/godot/commit/2930fbeecd7eab4ccffd4aea82964f2b8fd25e84, and thus confirmed that it's a 1:1 sync.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**